source/includes/write/transaction.php

```diff
@@ -0,0 +1,53 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// begin-callback
+$receipts = $client->bank->receipts;
+$checking = $client->bank->checking_accounts;
+$saving = $client->bank->saving_accounts;
+
+$accountId = '5678';
+$transferAmount = 1000.00;
+
+$callback = function (MongoDB\Driver\Session $session) 
+    use ($checking, $saving, $receipts, $accountId, $transferAmount): void {
+
+    $checking->updateOne(
+           ['account_id' => $accountId],
+           ['$inc' => ['balance' => -$transferAmount]],
+           ['session' => $session]
+    );
+    
+    $saving->updateOne(
+        ['account_id' => $accountId],
+        ['$inc' => ['balance' => $transferAmount]],
+        ['session' => $session]
+    );
+
+    $summary = sprintf(
+        "SAVINGS +%u CHECKING -%u.", $transferAmount, $transferAmount
+    );
+
+    $receipts->insertOne([
+        'account_id' => $accountId,
+        'description' => $summary,
+        'timestamp' => new MongoDB\BSON\UTCDateTime(),
+    ],
+    ['session' => $session]);
+
+    echo 'Successfully performed transaction!' , "\n";
```

Comment:
S: I've been using PHP_EOL instead of \n to avoid double quotes
```suggestion
    echo 'Successfully performed transaction!' , PHP_EOL;
```